docs/spec/liquidation-auction.rst

```diff
@@ -87,13 +89,9 @@ prefix and in front of the ``queued_slices``.
 Then we start an auction. An auction has minimum_bid value that is a
 function of current time and the latest bid.
 
-Every bid should be at least of ``minimum_bid`` amount of kit. Bidding
-process returns a bid token that can be sent back to claim either the
-bid itself when beaten, or the result of an auction (details will follow
-for this case).
-
-The latest bid always becomes ``leading_bid``. If a bid is not a leading
-bid of any current or past auction, they can be claimed back.
+Every bid should be of at least ``minimum_bid`` amount of kit. The bidding
+process debits the bid's kit from the contract's kit ledger and credits back
+the kit of the previously winning bid if one exists.
```

Comment:
```suggestion
the kit of the previously winning bid, if one exists.
```

Reply:
I'm not usually much of a grammar buff, but I ended up going down a bit of a rabbit hole on this topic since I was unsure which way was correct :sweat_smile:. I'm pretty sure without the comma is the correct way in this case:

https://english.stackexchange.com/questions/37856/do-you-need-a-comma-before-a-subordinating-conjunction-like-if
https://prowritingaid.com/grammar/1000080/Should-I-use-a-comma-before-a-subordinate-conjunction

Reply:
Hahaha, I didn't mean to get you into trouble! :stuck_out_tongue: I personally would keep the comma there because I'd also pause when reading the sentence but feel free to ignore the suggestion anyway, it's merely a suggestion :wink: